src/components/Publication/SinglePublication.tsx

```diff
@@ -35,7 +35,11 @@ const SinglePublication: FC<Props> = ({
   const { push } = useRouter();
   const isMirror = publication.__typename === 'Mirror';
   const firstComment = feedItem?.comments && feedItem.comments[0];
-  const rootPublication = feedItem ? (firstComment ? firstComment : feedItem?.root) : publication;
+  const rootPublication = feedItem
```

Comment:
Can we move to a function with a nice if condition? nested ternaries are difficult to read 🙏🏼